README.md

```diff
@@ -68,18 +68,89 @@ By following these steps, you should be able to install and run your application
 
 6. Once the containers are built and started, you can go inside the app container: `docker exec -it hff-hof-feedback-form-1 sh` (note: Docker containers may be named differently) or inspect the containers with Docker desktop.
 
-7. The volumes configured in `docker-compose.yml` should allow for hot-reloading of changes to files in the `/apps` and `/assets` folders. Changes to other root files may require a container rebuild with `docker-compose build`.
+7. The volumes configured in `docker-compose.yml` should allow for hot-reloading of changes to files in the `/apps`, `/assets` and `/utils` folders. Changes to other root level files may require a container rebuild with `docker-compose build` or at least to `docker-compose down` and then a fresh `docker-compose up`.
 
-### Testing
+## Testing
 
-#### Linting Tests
+### Linting Tests
 
-`$ yarn run test:lint`
+`$ yarn test:lint`
 
-#### Unit Tests
+### Unit Tests
 
 `$ yarn test:unit`
 
-### Deployment
+## Deployment
 
 This application is containerised and ready for deployment on Kubernetes. Refer to the `kube/` directory for Kubernetes deployment scripts.
+
+## Linking to this feedback form from another HOF project
+
+In a HOF project the feedback banner and URL are enabled by adding it to res.locals via an `app.use` function in the server.js of that project e.g.
+
+```javascript
+app.use((req, res, next) => {
+  res.locals.feedbackUrl = 'https://hof-feedback.homeoffice.gov.uk';
+  next();
+});
+```
```

Comment:
as discussed, the `config.js` file is the preferred place to define the `feedbackUrl'. So this section can be removed to avoid confusion